lang/de.js

```diff
@@ -18,10 +18,10 @@ export default {
 	"components.filter.clearDescription": "Filter für {filterName} löschen",
 	"components.filter.clearDescriptionSingle": "Filter löschen",
 	"components.filter.loading": "Filter werden geladen",
-	"components.filter.filterCountDescription": "{number, plural, =0 {Keine Filter angewendet.} one {1 Filter angewendet.} other {{number} Filter angewendet.}}",
+	"components.filter.filterCountDescription": "{number, plural, =0 {Keine Filter angewendet.} one {1 Filter angewendet.} other {{number} Filter angewendet.}}",
```

Comment:
I'm not sure what actually changed here... both seem to be the ASCII character `32` (space). 😕

Reply:
When I do the look-up, it appears they updated to use ASCII character 160 (U+00A0).  Apparently this is a non-breaking space, according to [Latin-1 Supplement](http://www.unicode.org/charts/PDF/U0080.pdf).

Reply:
Yikes, so we probably don't want this change then.

Reply:
This is correct. Some languages (honestly, probably true in English as well) they don't want the number separated from its subject if it happens to wrap in just the right spot.